scripts/check_frontend_coverage_test.py

```diff
@@ -281,19 +269,14 @@ def test_fully_covered_filenames_is_sorted(self):
         )
         fully_covered_tests_swap = self.swap(check_frontend_coverage,
             'fully_covered_filenames', [
-                'file.ts',
                 'anotherfile.ts'
-            ]
-        )
+                'file.ts',
+            ])
 
         with self.exists_swap, self.open_file_swap, self.print_swap:
             with fully_covered_tests_swap:
-                with self.assertRaises(SystemExit) as error:          
-                    check_frontend_coverage.check_coverage_changes()
-                self.assertEqual(error.exception.code, 1)
-                self.assertNotEqual(self.print_messages.index(
-                    ('The \033[1mfully_covered_filenames\033[0m list must be'
-                    ' kept in alphabetical order.')), -1)
+                (check_frontend_coverage
+                    .check_fully_covered_filenames_list_is_sorted())
```

Comment:
There needs to be an assertion in the test.

Reply:
Done.